src/lib/sql/functions.sql

```diff
@@ -8,9 +8,34 @@ SELECT
     ELSE pg_get_functiondef(p.oid)
   END AS definition,
   pg_get_function_arguments(p.oid) AS argument_types,
-  t.typname AS return_type
+  t.typname AS return_type,
+  CASE
+    WHEN p.provolatile = 'i' THEN 'IMMUTABLE'
+    WHEN p.provolatile = 's' THEN 'STABLE'
+    WHEN p.provolatile = 'v' THEN 'VOLATILE'
+  END AS behavior,
+  p.prosecdef AS security_definer,
+  JSON_OBJECT_AGG(p_config.param, p_config.values)
+    FILTER (WHERE p_config.param IS NOT NULL) AS config_params
 FROM
   pg_proc p
   LEFT JOIN pg_namespace n ON p.pronamespace = n.oid
   LEFT JOIN pg_language l ON p.prolang = l.oid
   LEFT JOIN pg_type t ON t.oid = p.prorettype
+  LEFT JOIN (
+    SELECT
+	    oid as id,
+	    (string_to_array(unnest(proconfig), '='))[1] AS param,
+   	  string_to_array((string_to_array(unnest(proconfig), '='))[2], ', ') AS values
+	  FROM
+	    pg_proc
+	) p_config ON p_config.id = p.oid
```

Comment:
@soedirgo not sure why the format here is so weird. Looks fine on my local:

![Screen Shot 2021-07-20 at 3 14 52 AM](https://user-images.githubusercontent.com/5532241/126277974-8db7438f-7339-4a60-871b-11112c55f7a2.png)

Reply:
Ah, looks like some tabs starting from line 27. Not a big deal.